tests/SampleApp/enc/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@
 
 oeedl_file(../SampleApp.edl enclave gen)
 
-add_enclave(TARGET SampleApp SOURCES SampleApp.cpp ${gen})
+add_enclave(TARGET SampleApp UUID 25419627-14f6-4625-9329-cf5f10a57fea SOURCES SampleApp.cpp ${gen})
```

Comment:
Can we leave an explanation in a local readme as to what the UUID here is for?

Reply:
Sure. We'll have to overhaul the documentation in any case to cover TrustZone, so this will be in that changeset, once we reach that point.